sites/dotnet-guide/contents/tutorials/rider-essentials/debugging/index.md

```diff
@@ -0,0 +1,51 @@
+---
+type: TutorialStep
+date: 2020-06-05
+title: Debugging code with Rider
+technologies: [.net, csharp]
+topics: [debugging]
+products: [rider]
+author: khalidabuhakmeh
+subtitle: Examine the runtime behavior of your application, identify problematic code, run through code step by step.
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://youtu.be/TODO
+---
+
+### Step through Code, Inspect what is Going On
```

Comment:
The capitalization here is inconsistent (at least to me)

Reply:
Better?

```suggestion
### Step Through Code, Inspect What is Going On
```